### ISSUE TITLE
nvim-hs-2.1.0.4: Expecting one more argument to ‘TyVarBndr’

### ISSUE THREAD

**Issue by juhp** [neovimhaskell/nvim-hs#90]
```
Building library for nvim-hs-2.1.0.4..                                                                                              
[17 of 28] Compiling Neovim.API.TH                                                                                                  
                                                                                                                                    
/var/stackage/work/unpack-dir/unpacked/nvim-hs-2.1.0.4-6a4f6b97674d67fe0d330805f11bbd81f7b7ef9ebb11cfd9beaeb637ee5ebb70/library/Neov
im/API/TH.hs:66:28: error:
    • Expecting one more argument to ‘TyVarBndr’
      Expected a type, but ‘TyVarBndr’ has kind ‘* -> *’
    • In the type signature:
        dataD' :: CxtQ -> Name -> [TyVarBndr] -> [ConQ] -> [Name] -> DecQ
   |
66 | dataD' :: CxtQ -> Name -> [TyVarBndr] -> [ConQ] -> [Name] -> DecQ
   |                            ^^^^^^^^^
```

**Comment by saep** [comment]
It would be helpful to know the stackage or ghc version used for compilation.

I suspect that the current code generation library `template-haskell` has changed. I'll have a look at this later today.

**Comment by saep** [comment]
This should be fixed with version 2.1.0.5.

I've also updated the dependency constraint for version 2.1.0.4 with a revision update, so cabal shouldn't pick this version anymore. If there are issues with the new version, feel free to open an issue again.
